### PR TITLE
chore: Bump inference and set default ocr_mode to entire_page

### PR DESCRIPTION
**Pull request by awalker4**
* pip-compile in order to bump unstructured-inference
* Set the default `ocr_mode` back to `enitre_page` now that [this error](https://github.com/Unstructured-IO/unstructured-inference/pull/183) is addressed
* Explicitly add `sphinx-tabs` to `build.in`. This file provides `docs/requirements.txt`.
* Remove a pinned `pydantic` version
* Fix a makefile command to `pip-compile` a missing ingest file.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 
 ### Enhancements
 * Create new CI Pipelines
   - Checking text, xml, email, and html doc tests against the library installed without extras
   - Checking each library extra against their respective tests
-* `partition` raises and error and tells the user to install the appropriate extra if a filetype
+* `partition` raises an error and tells the user to install the appropriate extra if a filetype
   is detected that is missing dependencies.
 * Add custom errors to ingest
+* Bump `unstructured-ingest` to 0.5.15
```
Comment (cragwolfe) [inline CHANGELOG.md:10]: please indicate unstructured-inference changes in sub-bullets.
e.g.:

https://github.com/Unstructured-IO/unstructured/blob/106ee965a6d072fa399e255a98fd761476e490f1/CHANGELOG.md?plain=1#L37-L39

File: requirements/constraints.in [hunk 2]
```diff
@@ -26,4 +26,6 @@ Pillow<10.0.0
 # AttributeError: 'ResourcePath' object has no attribute 'collection'
 Office365-REST-Python-Client<2.4.3
 # NOTE(christine) Pinned to set the `unstructured-inference` version
-unstructured-inference==0.5.13
\ No newline at end of file
+unstructured-inference==0.5.15
+# NOTE(crag) pinning to avoid error in argilla library
```
Comment (awalker4) [inline requirements/constraints.in:30]: FYI @cragwolfe I think you pinned this, right? I put your name for context.

Comment (cragwolfe) [reply requirements/constraints.in:30]: ah, that was for pydantic below i think. it's also been quite awhile. i.e., not related to unstructured-inference. is it causing issues?

**Comment by awalker4** [comment]
cc @jackretterer @cragwolfe 

The original `pip-compile` here removed `sphinx-tabs`. I added it back in to `build.in` which puts the package into `docs/requirements.txt` alongside some other sphinx stuff. Let me know if this looks right.

**Pull request merged**